docker-storage-setup.conf

```diff
@@ -1,20 +1,21 @@
-# Specify storage driver one wants to use with docker. Default is devicemapper.
+# Specify storage driver one wants to use with container runtimes.
+# Default is devicemapper.
 # Other possible options are overlay, overlay2 and "". Empty string means do
 # not do any storage setup.
 STORAGE_DRIVER=devicemapper
 
-# A set of extra options that should be appended to the generated
-# DOCKER_STORAGE_OPTIONS string. These options will be passed to the Docker
-# daemon as-is and should be valid Docker storage options.
-# EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=ext4"
+# Set extra options that will be appended to the generated STORAGE_OPTIONS
```

Comment:
Yes once you are happy with the man page, I will put this patch into the other patch set and close this down,  Then we can make sure docker-storage-setup actually matches the man page.

We will not merge this patch until Code is generating STORAGE_OPTIONS.